packages/radio/src/mwc-radio-base.ts

```diff
@@ -24,11 +24,46 @@ export class RadioBase extends FormElement {
 
   @query('input') protected formElement!: HTMLInputElement;
 
-  @property({type: Boolean})
-  @observer(function(this: RadioBase, checked: boolean) {
-    this.formElement.checked = checked;
-  })
-  checked = false;
+  private _checked = false;
```

Comment:
Should this be `protected`?

Reply:
Discussed offline, private makes sense for now.